0000-Install-and-Test/NodeRED-and-TJBot-Nodes/soict_s4v/tests/test.root.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2016-2018 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var sleep = require('sleep');
+
+const TJBot = require('tjbot');
+
+var tj = new TJBot(['servo'], {log: {level: 'debug'}}, {});
+
+console.log("Giơ tay ra sau");
+tj.armBack();
+sleep.sleep(1); //
+
+console.log("Giơ tay lên trên");
+tj.raiseArm();
+sleep.sleep(1); //
+
+console.log("Giơ tay phía trước ");
+tj.lowerArm();
+sleep.sleep(1); 
+
+console.log("Giơ tay ngang ra sau");
+tj.armBack();
+sleep.sleep(1); 
```

Comment:
Imho, I expect the final state of this test is "lower Arm" or originial state. It is better for calibration process.

Reply:
Ok. Right.
But I make sure that the first and the last position should be the same.